R/get_oaproc.R

```diff
@@ -175,10 +174,13 @@ get_xs_point <- function(point, width, num_pts) {
 #'
 #' }
 #'
-get_xs_points <- function(point1, point2, num_pts, res = 1) {
+get_xs_points <- function(points, num_pts, res = 1) {
 
-  point1 <- check_point(point1)[[1]]
-  point2 <- check_point(point2)[[1]]
+  points <- split(points, sort(as.numeric(rownames(points))))
```

Comment:
If this is supposed to be ordered already, let's avoid the sort here?

Reply:
Sounds good, i'll take it out.